app/graphql/types/query_type.rb

```diff
@@ -1,5 +1,20 @@
 module Types
   class QueryType < Types::BaseObject
+    # /cars
+    field :cars, [Types::CarType], null: false
+
+    def cars
+      Car.all
+    end
+
+    field :car, Types::CarType, null: false do
+      argument :id, ID, required: true
+    end
+
+    def car(id:)
+      Car.find(id)
+    end
```

Comment:
Awesome work! Should these methods live elsewhere though? Not saying they have to, genuinely just not sure how this should be structured